test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExe.cs

```diff
@@ -657,7 +657,7 @@ static void Main(string[] args)
                 .And.NotHaveStdOutContaining("Could not determine");
         }
 
-        [FullMSBuildOnlyTheory(Skip = "https://github.com/NuGet/Home/issues/8238")]
+        [FullMSBuildOnlyTheory]
```

Comment:
This test still fails. When it tries to build the test project, it errors with:
```
C:\h\w\BA370A1F\t\dotnetSdkTests\4g0jldi5.t0a\DesktopInBoxC---C644200B\DesktopInBoxConflictResolution\Program.cs(1,14): error CS0430: The extern alias 'snh' was not specified in a /reference option [C:\h\w\BA370A1F\t\dotnetSdkTests\4g0jldi5.t0a\DesktopInBoxC---C644200B\DesktopInBoxConflictResolution\DesktopInBoxConflictResolution.csproj]
```

The original issue was closed and copied to a new issue. So, I'll change this Skip text to be against the new issue.